seacrowd/sea_datasets/filwordnet/filwordnet.py

```diff
@@ -0,0 +1,137 @@
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class FilWordNet(datasets.GeneratorBasedBuilder):
+    """The Filipino WordNet (FilWordNet) is a lexical database of Filipino language containing 13,539 unique words and 9,519 synsets."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        )
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            # TODO: Create your source schema here
+            features = datasets.Features(
+                {
+                    "word_id": datasets.Value("int32"),
+                    "lemma": datasets.Value("string"),
+                    "synset_id": datasets.Value("int32"),
+                    "sense_id": datasets.Value("int32"),
+                    "pos": datasets.Value("string"),
+                    "lexdomain_id": datasets.Value("int32"),
+                    "definition": datasets.Value("string"),
+                    "last_modifier": datasets.Value("int32"),
+                    "sumo": datasets.Value("string"),
+                }
+            )
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        urls = _URLS[_DATASETNAME]
+        file = dl_manager.download_and_extract(urls)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": file,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        rows = []
+        is_first_row = True
+        with open(filepath, "r") as file:
+            csv_reader = csv.reader(file, delimiter=",")
+            for row in csv_reader:
+                if is_first_row:  # skip first row, they are column names
+                    is_first_row = False
+                    continue
+
+                rows.append(row)
+
+        if self.config.schema == "source":
+            for key, row in enumerate(rows):
+                example = {"word_id": row[0], "lemma": row[1], "synset_id": row[2], "sense_id": row[3], "pos": row[4], "lexdomain_id": row[5], "definition": row[6], "last_modifier": row[7], "sumo": row[8]}
```

Comment:
nit: This line may be too long. One technique I do is to add a comma at the end (i.e., at the end of `"sumo": row[8]`) and let the formatter arrange it in a readable format.